src/main/java/uk/co/onsdigital/discovery/model/HierarchyEntry.java

```diff
@@ -12,44 +12,38 @@
 import javax.persistence.UniqueConstraint;
 import java.io.Serializable;
 import java.util.List;
+import java.util.UUID;
 
 /**
  * The persistent class for the hierarchy_entry database table.
  */
 @Entity
-@Table(name = "hierarchy_entry", uniqueConstraints=@UniqueConstraint(columnNames={"hierarchy_id", "value_code"}))
-@IdClass(HierarchyEntry.HierarchyEntryId.class)
+@Table(name = "hierarchy_entry", uniqueConstraints = @UniqueConstraint(columnNames={"hierarchy_id", "code"}))
 public class HierarchyEntry {
 
     private static final long serialVersionUID = 1L;
 
     @Id
-    @Column(name = "hierarchy_id")
-    private String hierarchyId;
+    private UUID id;
```

Comment:
Looking at another UUID column definition, it specifies the columnDefinition as "uuid". Do we need this?:
@Column(name = "id", columnDefinition = "uuid")